src/order.cpp

```diff
@@ -3287,30 +3297,26 @@ static inline RtrBestResult decideWhereToRepairAndBalance(DROID *psDroid)
 			}
 		}
 	}
-	// if we are repair droid ourselves, don't consider other repairs droids
-	// because that causes havoc on front line: RT repairing themselves,
-	// blocking everyone else. And everyone else moving toward RT, also toward front line.s
-	// Ideally, we should just avoid retreating toward "danger", but dangerMap is only for multiplayer
-	if (psDroid->droidType != DROID_REPAIR && psDroid->droidType != DROID_CYBORG_REPAIR)
```

Comment:
@DARwins1: This functionality overall is great, but quick question in regards to this change: 

It looks like you removed a condition here that would prevent repair droids from retreating to other repair droids.

While the added functionality allows the user to configure whether repair droids accept retreating units (which is fantastic), won't the default behavior now be closer to the problematic behavior this condition tried to address (without modifying secondary unit orders - which some players don't use or even know exist - unless I'm missing something?)

Just wanted to discuss it.

Reply:
Repair droids now follow the same rules as any other retreating droid, it looks for the nearest repair facility or repair droid with this suborder enabled, while ignoring repair droids that have it disabled. If the player doesn't enable the suborder on any repair droids, then retreating repair droids (along with any other type of unit) will retreat to the nearest repair facility/command center as per usual.